factories/skins.rb

```diff
@@ -1,8 +1,12 @@
 require 'faker'
 FactoryBot.define do
+  sequence :skin_title do |n|
+    "#{Faker::Lorem.word} #{n}"
+  end
+
   factory :skin do
     author_id { FactoryBot.create(:user).id }
-    title { Faker::Lorem.word }
```

Comment:
FYI you can also use the [unique generator](https://github.com/faker-ruby/faker#ensuring-unique-values). No need to change this, though.